go.mod

```diff
@@ -92,7 +92,7 @@ require (
 	github.com/siderolabs/discovery-api v0.1.3
 	github.com/siderolabs/discovery-client v0.1.5
 	github.com/siderolabs/gen v0.4.5
-	github.com/siderolabs/go-blockdevice v0.4.5
+	github.com/siderolabs/go-blockdevice v0.4.6-0.20230704154634-fbb01f714bdc
```

Comment:
nit: it makes sense to release both go-blockdevice & kms-client before we merge this PR to establish base versions